basis_set_exchange/curate/metadata.py

```diff
@@ -84,6 +84,7 @@ def create_metadata_file(output_path, data_dir):
         # Create the common metadata for this basis set
         # display_name and other_names are placeholders to keep order
         # yapf: disable
+        print(base_filename)
```

Comment:
Should this be here?

Reply:
Nope. Can't believe I forgot that. I just removed it in master.